### ISSUE TITLE
Make Fishoids Level 13+ Easier

### ISSUE THREAD

**Issue by jerkyspace** [jessaskey/mhavocpe#104]
We talked about this at CAX 2019. The Fishoids are just too hard from Level 13+. If you don't line them up right at the start it's like a coin toss on whether you live or die. Sometimes one or two peel right off the circle formation when they are right above the player and kill you right away too.

They dive down too fast onto the player in general.

They too often jump over to the outsides of the screen and start looping around at the bottom right away giving the player no chance to react..

When they do loop around at the bottom they make these super fast tiny tight circles the player can't hope to get inside to shoot them from. Earlier in the game they loop in a much bigger circle slower so the player can get inside and shoot them.

I suggest slowing them down as far as how fast they go down the screen. Find a way to cut down on their tendency to go to the outside of the screen edges so much. When they do get past the player to the bottom make the looping much bigger and slower to give them a chance.

**Comment by jessaskey** [comment]
So I think I can do three things here that you have above or we have talked about in person

1. Slow them down a bit in general.
2. After Circling 3 times, have them go back up.
3. Widen the circles or limit how small the circle can get.

**Comment by jessaskey** [comment]
Okay, I made some code changes on this...  some general info before I go into what I changed. Firstly, the Fishoids level is configured to only have 4 stages of difficulty... Level 1, Level 5, Level 9 and Level 13... levels after 13 (17,21) are all the same difficulty of Level 13.

I made the Level 13+ stage of difficulty basically have the same 'speed' as Level 9 now. By 'Speed' Im talking about the animation speed of the whole space level... the way the original code was configured was that Level 5 was twice as fast as Level 1 and Level 9 was twice as fast as Level 5 and Level 13 was twice as fast as Level 9. Now, it caps out at the Level 9 speed.

After circling 3 times, I was attempting to have the Fishoid go back up to the top and then turn around and drop+track the player again like normal, however, I ended up with a bug in my new code that had an interesting side effect which is too hard to describe here, but it basically solves the problem and it looks sorta cool. In the end, the 'horizontal' death sweep that the fishoids used to do after the 3 loops is gone and now the continue to loop with with stutter turn in the middle.

I also was able to widen the loop diameters, the Level 13 stage was certainly the tightest with the fishoids pretty much doing a single loop, then a super tight loop for the remainders. I made Level 13 very similar to Level 9 now, but with one loop going slightly tighter than Level 9, so it is technically more difficult but very similar. 

On most of the above changes I can tweak those Level 13 metrics a bit to make them harder if needed again, just need feedback once this next ROM version is released (0.53).
